ui/components/MeshConfigComponent.js

```diff
@@ -270,6 +281,33 @@ class MeshConfigComponent extends React.Component {
     }, self.handleError);
   }
 
+  handleKubernetesClick = () => {
```

Comment:
surprised this is showing in as new. . .

Reply:
Even i am surprised, i guess i will have to make new one!